cocos/base/CCDirector.cpp

```diff
@@ -1076,7 +1076,7 @@ void Director::showStats()
     static unsigned long prevCalls = 0;
     static unsigned long prevVerts = 0;
     static float prevDeltaTime  = 0.016; // 60FPS
-    static const float FPS_FILTER = 0.05;
+    static const float FPS_FILTER = 0.10;
```

Comment:
I guess it just used for making FPS display more stable, right?
Ccurrently, I wanna get the real FPS (frameRate = 1/dt), but `_frameRate` will return a value after the calculation of `FPS_FILTER`.